db/migrate/20220115062051_add_last_fetched_to_feeds.rb

```diff
@@ -0,0 +1,6 @@
+class AddLastFetchedToFeeds < ActiveRecord::Migration[7.0]
+  def change
+    add_column :feeds, :last_fetched, :datetime
```

Comment:
convention names timestamp columns ending with _at, to match created_at and updated_at

Reply:
I guess with that knowledge combined with how `updated_at` already stores "the last time this was updated" we can rename `last_fetched_at` to just `fetched_at`